Source/SourceKittenFramework/SourceLocation.swift

```diff
@@ -24,7 +24,7 @@ public struct SourceLocation {
 
 extension SourceLocation {
     init(clangLocation: CXSourceLocation) {
-        var cxfile = CXFile.alloc(1)
+        var cxfile: CXFile? = .allocate(bytes: 1, alignedTo: 0)
```

Comment:
Yes, that works, though we may have to go back to this approach when we add Linux support. I'll change it for now.